### PR TITLE
Remove ee_brightbox integration

### PR DESCRIPTION
**Pull request by epenet**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/63839
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3085
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by ludeeus — CHANGES_REQUESTED** [review]
This needs a redirect like all other removed integrations

**Comment by epenet** [comment]
> This needs a redirect like all other removed integrations

Is this right?

**Review by ludeeus — DISMISSED** [review]

File: source/_redirects [hunk 1]
```diff
@@ -2355,3 +2355,4 @@
 /components/sensor.arduino /more-info/removed-integration 301
 /components/switch.arduino /more-info/removed-integration 301
 /integrations/arduino /more-info/removed-integration 301
+/integrations/ee_brightbox /more-info/removed-integration 301
```
Comment (frenck) [inline source/_redirects:2358]: `ee_brightbox` has more occurrences in this redirect file that needs to be moved to this location and redirect directly to the removed integration page.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_redirects (hunk 1 above)
Comment (epenet) [reply source/_redirects:2358]: This should be better

**Review by frenck — APPROVED** [review]

**Pull request merged**